README.md

```diff
@@ -62,6 +62,9 @@ When reading files the API accepts several options:
     * When it encounters a field of the wrong datatype, it sets the offending field to `null`.
   * `DROPMALFORMED` : ignores the whole corrupted records.
   * `FAILFAST` : throws an exception when it meets corrupted records.
+* `inferSchema`: Trying to infer column data type or not. Default is `true`.
```

Comment:
I would just write...
```
`inferSchema`: if `true`, attempts to infer an appropriate type for each resulting DataFrame column, like a boolean, numeric or date type. If `false`, all resulting columns are of string type.
```

Reply:
That's indeed more concise.
I'll update the description.